Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -275,20 +298,33 @@ public class HTTPServer: Server {
             }
             .childChannelOption(ChannelOptions.socket(IPPROTO_TCP, TCP_NODELAY), value: 1)
 
+        let listenerDescription: String
         do {
-            serverChannel = try bootstrap.bind(host: "0.0.0.0", port: port).wait()
+            switch socket {
+            case SocketType.tcp(let port):
+                listenerDescription = "port \(port)"
```

Comment:
I just realized this isn't quite right: this will result in `port 0` when listening on an ephemeral port.

Reply:
I've fixed this in https://github.com/IBM-Swift/Kitura-NIO/pull/187/commits/c7154c8fb7af29c13f8f9db4c4d417cca4187efc